.organizationmap

```diff
@@ -272,7 +272,10 @@ Nebula <info@nebula.com> Chris Holcombe <chris.holcombe@nebula.com>
 Ocado <contact@ocado.com> Luis Periquito <luis.periquito@ocado.com>
 Odiso <contact@odiso.com> Alexandre Derumier <aderumier@odiso.com>
 Opower <contact@opower.com> Derrick Schneider <derrick.schneider@opower.com>
-OVH <contact@corp.ovh.com> Etienne Menguy <etienne.menguy@corp.ovh.com>
+OVH <ceph@ovh.net> Etienne Menguy <etienne.menguy@corp.ovh.com>
+OVH <ceph@ovh.net> Piotr Dałek <piotr.dalek@corp.ovh.com>
```

Comment:
can you make an entry for Piotr in .peoplemap as well, this captures the change in affiliation when we generate credits so that older credits affiliate correctly as well